website/content/en/docs/tutorials/iperf.md

```diff
@@ -30,48 +29,50 @@ Before you start, you must have access to two Kubernetes clusters.
 For example, In this tutorial we set up a local environment using the [Kind](https://kind.sigs.k8s.io/) project.
 To setup two kind clusters:
 
-1. Install Kind project using [Kind installtion guide](https://kind.sigs.k8s.io/docs/user/quick-start).
-1. Create a test folder for all the tutorial files:
+1. Install Kind using [Kind installation guide](https://kind.sigs.k8s.io/docs/user/quick-start).
+2. Create a folder for all the tutorial files:
 
     ```sh
     mkdir iperf3-tutorial
     ```
 
-1. Open two terminals in the tutorial folder and create a Kind cluster in each terminal:
+3. Open two terminals in the tutorial folder and create a Kind cluster in each terminal:
 
     Client cluster:
 
     ```sh
     cd iperf3-tutorial
-    kind create cluster --name=peer1
+    kind create cluster --name=client
     ```
 
     Server cluster:
 
     ```sh
     cd iperf3-tutorial
-    kind create cluster --name=peer2
+    kind create cluster --name=server
     ```
 
```

Comment:
nit: for users not too familiar with kind, consider adding a note/tip that kind names clusters' context in the kubeconfig with prefix `kind-` so name would be different than specified on the command line?